changelogs/fragments/798-homebrew-cask-greedy-for-upgrades-too.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - homebrew_cask - consider ``greedy`` option for upgrade_all, too (https://github.com/ansible-collections/community.general/pull/798)
```

```suggestion
  - homebrew_cask - consider ``greedy`` option for upgrade_all, too (https://github.com/ansible-collections/community.general/pull/798).
```
Sorry, I totally missed that period...